packages/main/src/TabContainer.js

```diff
@@ -226,6 +227,46 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 * Defines the background color of the Tab Container's header.
```

Comment:
I suggest to change this description as:
* Sets the background color of the Tab Container's header as <code>Solid</code>, <code>Transparent</code>, or <code>Translucent</code>.